packages/@coorpacademy-components/src/organism/list-item/index.js

```diff
@@ -1,33 +1,58 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import {isEmpty, map} from 'lodash/fp';
+import {isEmpty, map, noop, get} from 'lodash/fp';
+import {convert} from 'css-color-function';
 import classnames from 'classnames';
+import Provider from '../../atom/provider';
 import Tag from '../../atom/tag';
+import Icon from '../../atom/icon';
 import ButtonLink from '../../atom/button-link';
 import BulletPointMenuButton from '../../molecule/bullet-point-menu-button';
 // eslint-disable-next-line css-modules/no-unused-class
 import style from './style.css';
 
-const ListItem = ({
-  buttonLink,
-  secondButtonLink,
-  bulletPointMenuButton,
-  tags,
-  title,
-  dataColumns,
-  order,
-  'aria-label': ariaLabel,
-  contentType,
-  isBulkStyle = false,
-  isOverflowHidden = false
-}) => {
+const ListItem = (
+  {
+    buttonLink,
+    secondButtonLink,
+    bulletPointMenuButton,
+    tags,
+    title,
+    selected,
+    subtitle,
+    dataColumns,
+    order,
+    'aria-label': ariaLabel,
+    disabled = false,
+    contentType,
+    isBulkStyle = false,
+    isOverflowHidden = false,
+    onClick = noop
+  },
+  context
+) => {
+  const {skin} = context;
+  const primarySkinColor = get('common.primary', skin);
   let isPublished = false;
 
+  const selectedStyle = selected
+    ? {
+        backgroundColor: convert(`color(${primarySkinColor} a(0.07))`)
+      }
+    : {};
+
+  const tagSelectedStyle = selected
+    ? {
+        backgroundColor: convert(`color(${primarySkinColor} a(0.15))`),
+        color: primarySkinColor
+      }
+    : {};
+
   const tagsView = map.convert({cap: false})((tag, index) => {
```

Comment:
(i no it's existing but can you extract a uncappedMap const as we usually do?

Reply:
actually, this is needed to get the index for the key param.

Reply:
I mean, extract a const so we dont do a convert over and over :)
cf https://github.com/CoorpAcademy/coorpacademy/blob/fee2e62c7dad94e79f2c59fe62b854a417f2aa13/core/lib/translate/content.js#L6